README.md

```diff
@@ -86,7 +95,9 @@ Services and operations managed or affected by
 [pupmod-simp-rsyslog](https://github.com/simp/pupmod-simp-rsyslog):
 * rsyslogd
 * auditd (configurable)
-* iptables (configurable)
+* firewall (configurable)
+  * NOTE: If firewall management is enabled, and you are using iptables (not
+    firewalld), then you MUST set ``iptables::precise_match: true`` in Hiera.
```

Comment:
Why is this true?  Is this new behavior that needs to be documented in simp-doc, since this the parameter defaults to false?

Reply:
Because if you switch from `127.0.0.1` (the default) to `ALL`, or something else that only changes a single rule, then `iptables` will not restart.